pkg/kubectl/cmd/delete.go

```diff
@@ -116,6 +116,7 @@ func RunDelete(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []str
 		SelectAllParam(deleteAll).
 		ResourceTypeOrNameArgs(false, args...).RequireObject(false).
 		Flatten().
+		Latest().
```

Comment:
FWIW, by default (i.e., cascade=true) many resources are deleted via a reaper, which does a GET and then a DELETE, e.g., https://github.com/kubernetes/kubernetes/blob/8c60068b368f8ec1cd21fdf25d44c7b165aa9d27/pkg/kubectl/stop.go#L435, do you think that's a violation?